shim.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	vc "github.com/containers/virtcontainers"
+)
+
+// ShimInfo gathers information needed by the shim.
+type ShimInfo struct {
+	Path  string
+	Token string
+	IP    string
+	Port  string
+}
+
+// Specific values related to the shim.
+const (
+	defaultShimPath  = "/usr/local/libexec/cc-shim"
```

Comment:
I agree about the follow up patch, but let's do that after we get this PR merged.